docs/user-guide/kubeadm-reference.md

```diff
@@ -0,0 +1,144 @@
+# kubeadm reference
+
+This document specifies the advanced options of kubeadm.
```

Comment:
Doc nit: "This document provides information on how to use kubeadm's advanced options."

("Specifying" is something the user does when they set a parameter, for example)